src/basilisp/core.lpy

```diff
@@ -1321,11 +1321,6 @@
   [x]
   (and (integer? x) (neg? x)))
 
-(defn  ^:inline nil?
-  "Return ``true`` if ``x`` is ``nil``\\, otherwise ``false``\\."
-  [x]
-  (operator/is- x nil))
```

Comment:
This was moved up to an earlier point in the file, but the original was never removed.